tools/bazar/fields/ImageField.php

```diff
@@ -17,12 +17,14 @@ class ImageField extends FileField
     protected $imageHeight;
     protected $imageWidth;
     protected $imageClass;
+    protected $imageDefault;
 
     protected const FIELD_THUMBNAIL_HEIGHT = 3;
     protected const FIELD_THUMBNAIL_WIDTH = 4;
     protected const FIELD_IMAGE_HEIGHT = 5;
     protected const FIELD_IMAGE_WIDTH = 6;
     protected const FIELD_IMAGE_CLASS = 7;
+    protected const FIELD_IMAGE_DEFAULT = 8;
```

Comment:
```suggestion
    protected const FIELD_IMAGE_DEFAULT = 13;
```
Il faudrait éviter d'utiliser le numéro 8 car il est utilisé `BazarField::FIELD_REQUIRED`. Le champ 13 semble disponible pour image (on ne peut pas dépasser 14 et les autres numéros semblent dékà pris pour Image)